### PR TITLE
Enhancements & Bug fixes

### PR DESCRIPTION
**Pull request by gibran96**
- Navbar UI responsiveness. I think overall all the pages are done, but still please go through the app and check if anything is breaking.
- Added snackbars and fixed autohide.
- Error page in case of 500 errors.
- Cart page empty cart reducer synced.
- Wishlist bugs fixed

### CODE REVIEW

File: src/Ekart/Cart/card-summary.js [hunk 1]
```diff
@@ -1,15 +1,28 @@
...
+import {Button, Card, CardContent, Typography} from "@mui/material";
+import React, {useState} from "react";
+import {emptyCart} from "./service";
+import {createOrder} from "../Orders/service";
+import {useNavigate} from "react-router-dom";
+import {useDispatch} from "react-redux";
+import {setCartItems} from "./cartReducer";
+import SnackbarComponent from "../../Common/snackbar";
 
-function CardSummary({ cartItems, setCartItems }) {
```
Comment (anantmoudgalya) [inline src/Ekart/Cart/card-summary.js:7]: removed `setCartItems` but it is used on line 18
I'm confused

Comment (gibran96) [reply src/Ekart/Cart/card-summary.js:7]: We dont need to pass the reducer method around, we can just import it in the card summary component

**Review by anantmoudgalya — APPROVED** [review]
LGTM except comment

File: src/Ekart/Cart/card-summary.js (hunk 1 above)
Comment (anantmoudgalya) [reply src/Ekart/Cart/card-summary.js:7]: ohh makes sense

**Pull request merged**